main.md

```diff
@@ -911,12 +916,6 @@ a nonce value to include in the DPoP proof:
 !---
 Figure: HTTP 400 Response to a Token Request without a Nonce
 
-Other HTTP headers and JSON fields MAY also be included in the error response, 
-but there MUST NOT be more than one `DPoP-Nonce` header.
-
-Upon receiving the nonce, the client is expected to retry its token request
-using a DPoP proof including the supplied nonce value in the `nonce` claim
-of the DPoP proof.
```

Comment:
All the deleted information above is useful.  I don't see it having been replaced elsewhere.  Please restore it.